pyam/__init__.py

```diff
@@ -13,21 +11,21 @@
     require_variable,
     validate,
 )
-from pyam.datareader import read_worldbank  # noqa: F401  # noqa: F401
 from pyam.iiasa import lazy_read_iiasa, read_iiasa  # noqa: F401  # noqa: F401
 from pyam.ixmp4 import read_ixmp4  # noqa: F401
+from pyam.logging import configure_logging  # noqa: F401
 from pyam.run_control import run_control  # noqa: F401
-from pyam.statistics import Statistics
+from pyam.statistics import Statistics  # noqa: F401
```

Comment:
In pyproject.toml, we have two lines that read:
```toml
[tool.ruff.lint.per-file-ignores]
"__init__.py" = ["F401"]
```
This should make it unnecessary to manually specify `# noqa: F401` in `__init__.py` files. Is there any specific reason to write it? A tool like mypy would not read this configuration setting, so requires its own setting, but ruff should not complain. If something like flake8 is still running in the background of your editor, feel free to switch it off since ruff provides the same functionality. If ruff itself is complaining, we should check its docs if the configuration syntax has changed for the version we're using.

Reply:
Done